src/cli.c

```diff
@@ -447,6 +450,33 @@ main(int argc, char** argv)
                output_format = MANAGEMENT_OUTPUT_FORMAT_TEXT;
             }
             break;
+         case 'C':
+            if (!strncmp(optarg, "gz", MISC_LENGTH))
+            {
+               compression = MANAGEMENT_COMPRESSION_GZIP;
+            }
+            else if (!strncmp(optarg, "zstd", MISC_LENGTH))
+            {
+               compression = MANAGEMENT_COMPRESSION_ZSTD;
+            }
+            else if (!strncmp(optarg, "lz4", MISC_LENGTH))
+            {
+               compression = MANAGEMENT_COMPRESSION_LZ4;
+            }
+            else if (!strncmp(optarg, "bz2", MISC_LENGTH))
+            {
+               compression = MANAGEMENT_COMPRESSION_BZIP2;
+            }
+            else if (!strncmp(optarg, "none", MISC_LENGTH))
+            {
+               break;
+            }
+            else
+            {
+               warnx("pgmoneta-cli: Use either -c or -h/-p to define endpoint");
```

Comment:
This is wrong

Reply:
There have been some changes to the `-F` option, which caused a conflict with my commit, leading to unintended changes.